js/dailyshine.js

```diff
@@ -141,6 +141,7 @@
         if (i == messages.length - 1) {
           data.linkTitle = localized(content.linkTitle);
           data.linkUrl = localized(content.linkUrl);
+          data.dateClicked = formatDateMMddyyy(new Date());
```

Comment:
Because people will have the ability to view content from previous days, instead of just using `new Date()` here, it should probably be something more like...

`getParameter('date') ? createDateFromQuery(getParameter('date')) : new Date()`